addons/api/addon/serializers/target.js

```diff
@@ -44,6 +44,14 @@ export default class TargetSerializer extends ApplicationSerializer {
         injectedApplicationCredentialSourceIDs,
       );
 
+    if (serialized?.with_aliases) {
+      // API expects scope id along with every alias value
+      serialized.with_aliases = serialized.with_aliases.map((item) => ({
+        ...item,
+        scope_id: 'global',
```

Comment:
None blocking comment: In the future, I guess we should pass a dynamic value as `scope_id`?

Reply:
Yup, when we have more than a global scope, we should allow the user to select the scope that they want to create in the UI, and we can pass in dynamically then. Right now, it is only global scope, hence I thought additional calls weren't necessary